eth/handler.go

```diff
@@ -173,7 +173,7 @@ func NewProtocolManager(config *params.ChainConfig, fastSync bool, networkId int
 		return blockchain.CurrentBlock().NumberU64()
 	}
 	inserter := func(blocks types.Blocks) (int, error) {
-		manager.setSynced() // Mark initial sync done on any fetcher import
+		atomic.StoreUint32(&manager.synced, 1) // Mark initial sync done on any fetcher import
```

Comment:
It doesn't hurt to use `atomic` here, but I'm curious... Previously, you actually _did_ something if `synched` went from `0` to `1`, and thus wanted it to be threadsafe/atomic. Now it seems you could just aswell use non-atomic, right?